src/Renderers/TitleNodeRenderer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Docs Builder package.
+ * (c) Ryan Weaver <ryan@symfonycasts.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SymfonyDocsBuilder\Renderers;
+
+use Doctrine\RST\Environment;
+use Doctrine\RST\Nodes\TitleNode;
+use Doctrine\RST\Renderers\NodeRenderer;
+use Doctrine\RST\Templates\TemplateRenderer;
+
+class TitleNodeRenderer implements NodeRenderer
+{
+    /** @var TitleNode */
+    private $titleNode;
```

Comment:
@javiereguiluz can we use typed properties?

Reply:
Let's check that ... but do it in other PR so we can merge and deploy this one as soon as possible. Thanks.